test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -208,8 +193,7 @@ var s_ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 					Expect(result).To(Equal("0"))
 				}
 			},
-			Entry("[test_id:68385] Verify ovs cgroup with crun ", true, true, true, true),
-			Entry("[test_id:64098] Verify ovs cgroup with runc", false, true, true, true),
+			Entry("[test_id:64098] Verify ovs cgroup with runc", true, true, true),
```

Comment:
If we only have a single entry let's just run it as regular `It`

Reply:
Addressed in the latest commit